inc/class-s3-uploads-wp-cli-command.php

```diff
@@ -3,6 +3,60 @@
 class S3_Uploads_WP_CLI_Command extends WP_CLI_Command {
 
 	/**
+	 * Verifies the API keys entered will work for writing and deleting from S3.
+	 *
+	 * @subcommand verify
+	 */
+    public function verify_api_keys() {
+
+		S3_Uploads::get_instance(); // Boot
```

Comment:
This should already be hooked on the normal plugin includes, or is there some other reason why this is needed?